examples/async-hello-world/async-hello-world.go

```diff
@@ -0,0 +1,142 @@
+/* This file is part of VoltDB.
```

Comment:
By naming convention, this file should be "async_hello_world.go" , right ? (underscore instead of hyphen)